### ISSUE TITLE
kubeadm jobs failing due to an old kubetest image

### ISSUE THREAD

**Issue by luxas** [kubernetes/test-infra#3835]
```
W0728 14:38:30.533] Run: ('kubetest', '-v', '--dump=/workspace/k8s.io/kubernetes/_artifacts', '--gcp-service-account=/etc/service-account/service-account.json', '--up', '--down', '--test', '--cluster=e2e-97', '--gcp-network=e2e-97', '--deployment=kubernetes-anywhere', '--extract=ci/latest-1.7', '--gcp-project=k8s-jkns-gci-gke-reboot-1-4', '--gcp-zone=us-central1-f', '--kubernetes-anywhere-kubernetes-version=latest-1.7', '--provider=kubernetes-anywhere', '--test_args=--ginkgo.focus=\\[Conformance\\] --minStartupPods=8', '--timeout=300m', '--kubernetes-anywhere-path=/workspace/kubernetes-anywhere', '--kubernetes-anywhere-phase2-provider=kubeadm', '--kubernetes-anywhere-cluster=e2e-97', '--kubernetes-anywhere-kubeadm-version=gs://kubernetes-release-dev/bazel/v1.7.3-beta.0.39+5387acecbb40d2/bin/linux/amd64/')
W0728 14:38:30.703] flag provided but not defined: -cluster
W0728 14:38:30.704] Usage of kubetest:
W0728 14:38:30.704]   -build
W0728 14:38:30.704]     	Rebuild k8s binaries, optionally forcing (release|quick|bazel) stategy
W0728 14:38:30.704]   -charts
W0728 14:38:30.704]     	If true, run charts tests
W0728 14:38:30.705]   -check-leaked-resources
W0728 14:38:30.705]     	Ensure project ends with the same resources
W0728 14:38:30.712]   -check-version-skew
W0728 14:38:30.712]     	Verify client and server versions match (default true)
W0728 14:38:30.712]   -deployment string
W0728 14:38:30.712]     	Choices: none/bash/kops/kubernetes-anywhere (default "bash")
W0728 14:38:30.712]   -down
W0728 14:38:30.713]     	If true, tear down the cluster before exiting.
W0728 14:38:30.713]   -dump string
W0728 14:38:30.713]     	If set, dump cluster logs to this location on test or cluster-up failure
W0728 14:38:30.713]   -extract value
W0728 14:38:30.713]     	Extract k8s binaries from the specified release location
W0728 14:38:30.714]   -federation
W0728 14:38:30.714]     	If true, start/tear down the federation control plane along with the clusters. To only start/tear down the federation control plane, specify --deploy=none
W0728 14:38:30.714]   -gcp-cloud-sdk string
W0728 14:38:30.714]     	Install/upgrade google-cloud-sdk to the gs:// path if set
W0728 14:38:30.714]   -gcp-project string
W0728 14:38:30.714]     	For use with gcloud commands
W0728 14:38:30.715]   -gcp-service-account string
W0728 14:38:30.715]     	Service account to activate before using gcloud
W0728 14:38:30.715]   -gcp-zone string
W0728 14:38:30.715]     	For use with gcloud commands
W0728 14:38:30.715]   -kops string
W0728 14:38:30.716]     	(kops only) Path to the kops binary. Must be set for kops.
W0728 14:38:30.716]   -kops-admin-access string
W0728 14:38:30.716]     	(kops only) If set, restrict apiserver access to this CIDR range.
W0728 14:38:30.716]   -kops-args string
W0728 14:38:30.716]     	(kops only) Additional space-separated args to pass unvalidated to 'kops create cluster', e.g. '--kops-args="--dns private --node-size t2.micro"'
W0728 14:38:30.716]   -kops-cluster string
W0728 14:38:30.717]     	(kops only) Cluster name. Must be set for kops.
W0728 14:38:30.717]   -kops-image string
W0728 14:38:30.717]     	(kops only) Image (AMI) for nodes to use. (Defaults to kops default, a Debian image with a custom kubernetes kernel.)
W0728 14:38:30.717]   -kops-kubernetes-version string
W0728 14:38:30.717]     	(kops only) If set, the version of Kubernetes to deploy (can be a URL to a GCS path where the release is stored) (Defaults to kops default, latest stable release.).
W0728 14:38:30.718]   -kops-nodes int
W0728 14:38:30.718]     	(kops only) Number of nodes to create. (default 2)
W0728 14:38:30.718]   -kops-priority-path string
W0728 14:38:30.718]     	Insert into PATH if set
W0728 14:38:30.718]   -kops-ssh-key string
W0728 14:38:30.719]     	(kops only) Path to ssh key-pair for each node (defaults '~/.ssh/kube_aws_rsa' if unset.)
W0728 14:38:30.719]   -kops-state string
W0728 14:38:30.719]     	(kops only) s3:// path to kops state store. Must be set.
W0728 14:38:30.719]   -kops-up-timeout duration
W0728 14:38:30.719]     	(kops only) Time limit between 'kops config / kops update' and a response from the Kubernetes API. (default 20m0s)
W0728 14:38:30.719]   -kops-zones string
W0728 14:38:30.720]     	(kops AWS only) AWS zones for kops deployment, comma delimited. (default "us-west-2a")
W0728 14:38:30.720]   -kubemark
W0728 14:38:30.720]     	If true, run kubemark tests.
W0728 14:38:30.720]   -kubemark-master-size string
W0728 14:38:30.720]     	Kubemark master size
W0728 14:38:30.721]   -kubemark-nodes string
W0728 14:38:30.721]     	Number of kubemark nodes to start
W0728 14:38:30.721]   -kubernetes-anywhere-cluster string
W0728 14:38:30.721]     	(kubernetes-anywhere only) Cluster name. Must be set for kubernetes-anywhere.
W0728 14:38:30.721]   -kubernetes-anywhere-kubeadm-version string
W0728 14:38:30.721]     	(kubernetes-anywhere only) Version of kubeadm to use, if phase2-provider is kubeadm. May be "stable" or a gs:// link to a custom build. (default "stable")
W0728 14:38:30.722]   -kubernetes-anywhere-kubernetes-version string
W0728 14:38:30.722]     	(kubernetes-anywhere only) Version of Kubernetes to use (e.g. latest, stable, latest-1.6, 1.6.3, etc).
W0728 14:38:30.722]   -kubernetes-anywhere-path string
W0728 14:38:30.722]     	(kubernetes-anywhere only) Path to the kubernetes-anywhere directory. Must be set for kubernetes-anywhere.
W0728 14:38:30.722]   -kubernetes-anywhere-phase2-provider string
W0728 14:38:30.723]     	(kubernetes-anywhere only) Provider for phase2 bootstrapping. (Defaults to ignition). (default "ignition")
W0728 14:38:30.723]   -kubernetes-anywhere-up-timeout duration
W0728 14:38:30.723]     	(kubernetes-anywhere only) Time limit between starting a cluster and making a successful call to the Kubernetes API. (default 20m0s)
W0728 14:38:30.723]   -logexporter-gcs-path string
W0728 14:38:30.723]     	Path to the GCS artifacts directory to dump logs from nodes. Logexporter gets enabled if this is non-empty
W0728 14:38:30.724]   -metadata-sources string
W0728 14:38:30.724]     	Comma-seperated list of files inside ./artifacts to merge into metadata.json (default "images.json")
W0728 14:38:30.724]   -multiple-federations
W0728 14:38:30.724]     	If true, enable running multiple federation control planes in parallel
W0728 14:38:30.724]   -perf-tests
W0728 14:38:30.725]     	If true, run tests from perf-tests repo.
W0728 14:38:30.725]   -provider string
W0728 14:38:30.725]     	Kubernetes provider such as gce, gke, aws, etc
W0728 14:38:30.725]   -publish string
W0728 14:38:30.725]     	Publish version to the specified gs:// path on success
W0728 14:38:30.725]   -registry string
W0728 14:38:30.726]     	Push images to the specified docker registry (e.g. gcr.io/a-test-project)
W0728 14:38:30.726]   -save string
W0728 14:38:30.726]     	Save credentials to gs:// path on --up if set (or load from there if not --up)
W0728 14:38:30.726]   -skew
W0728 14:38:30.726]     	If true, run tests in another version at ../kubernetes/hack/e2e.go
W0728 14:38:30.726]   -stage value
W0728 14:38:30.727]     	Upload binaries to gs://bucket/devel/job-suffix if set
W0728 14:38:30.727]   -stage-suffix string
W0728 14:38:30.727]     	Append suffix to staged version when set
W0728 14:38:30.727]   -test
W0728 14:38:30.727]     	Run Ginkgo tests.
W0728 14:38:30.728]   -test_args string
W0728 14:38:30.728]     	Space-separated list of arguments to pass to Ginkgo test runner.
W0728 14:38:30.728]   -timeout duration
W0728 14:38:30.728]     	Terminate testing after the timeout duration (s/m/h)
W0728 14:38:30.728]   -up
W0728 14:38:30.728]     	If true, start the the e2e cluster. If cluster is already up, recreate it.
W0728 14:38:30.729]   -upgrade_args string
W0728 14:38:30.729]     	If set, run upgrade tests before other tests
W0728 14:38:30.729]   -v	If true, print all command output.
W0728 14:38:30.729] Traceback (most recent call last):
W0728 14:38:30.729]   File "/workspace/./test-infra/jenkins/../scenarios/kubernetes_e2e.py", line 522, in <module>
W0728 14:38:30.729]     main(parse_args())
W0728 14:38:30.730]   File "/workspace/./test-infra/jenkins/../scenarios/kubernetes_e2e.py", line 432, in main
W0728 14:38:30.730]     mode.start(runner_args)
W0728 14:38:30.730]   File "/workspace/./test-infra/jenkins/../scenarios/kubernetes_e2e.py", line 188, in start
W0728 14:38:30.730]     check_env(env, 'kubetest', *args)
W0728 14:38:30.730]   File "/workspace/./test-infra/jenkins/../scenarios/kubernetes_e2e.py", line 57, in check_env
W0728 14:38:30.731]     subprocess.check_call(cmd, env=env)
W0728 14:38:30.731]   File "/usr/lib/python2.7/subprocess.py", line 540, in check_call
W0728 14:38:30.731]     raise CalledProcessError(retcode, cmd)
```

@krzyzacy @pipejakob Seems like a similar thing as https://github.com/kubernetes/test-infra/pull/3752 is needed :'(

Real issue to fix long-term: https://github.com/kubernetes/test-infra/issues/3606

**Comment by luxas** [comment]
cc @wojtek-t @fejta 

Currently affects the https://k8s-testgrid.appspot.com/sig-cluster-lifecycle#kubeadm-gce-1.7 and https://k8s-testgrid.appspot.com/sig-cluster-lifecycle#kubeadm-gce-1.6-on-1.7 jobs.
Not sure if this is fixed already and prow/jenkins/etc just have to be reloaded

**Comment by krzyzacy** [comment]
@luxas that was before #3752 is merged, and http://prow.k8s.io/?type=periodic&job=periodic-kubernetes-bazel-test-1-7 the precedence job is broken so kubeadm is never triggered after July.28th

**Comment by krzyzacy** [comment]
cc @ixdy --> https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/logs/periodic-kubernetes-bazel-test-1-7/547/

**Comment by ixdy** [comment]
oh, yeah, I need to cherry-pick my fix to 1.7. lemme get on that.

**Comment by ixdy** [comment]
https://github.com/kubernetes/kubernetes/pull/49975
